cmd/modelcmd/base.go

```diff
@@ -539,7 +540,12 @@ func (g bootstrapConfigGetter) getBootstrapConfigParams(controllerName string) (
 			Credential:       credential,
 		},
 		Config: cfg,
-	}, nil
+	}
+	if cloudCACerts, ok := bootstrapConfig.ControllerConfig.CloudCACerts(); ok {
```

Comment:
If the Controller config was given certs (as per above) the we want to populate the `prepareCfgParams` with them. Looks good.